framework/web/UrlManager.php

```diff
@@ -414,6 +415,7 @@ public function createUrl($params)
                 $route .= '?' . $query;
             }
 
+            $route = ltrim($route, '/');
```

Comment:
This is not needed, `$route` is already trimmed: https://github.com/cebe/yii2/blob/9a53571fd2cac0155477bef8420a74b0b665f2c7/framework/web/UrlManager.php#L360